### ISSUE TITLE
Re-enable SSL Cert Verification

### ISSUE THREAD

**Issue by alexkyllo** [ScienceCommons/api#236]
bcoe turned off SSL Cert Verification with this commit: https://github.com/ScienceCommons/api/commit/4fb4cccabf99eeb70f3a38ba1dab2c0045a0086f#commitcomment-13273262

Need to turn it back on as this is a security hole.

**Comment by alexkyllo** [comment]
@eplebel I pushed a fix for this issue to staging. It works for me but please verify that you can still log into staging with your google account and do not get a certificate verification error. If it works for you I will merge into master.

**Comment by eplebel** [comment]
yup i was able to logout and log back in with my google account no problem!

**Comment by alexkyllo** [comment]
Search breaks in staging but not in production if we turn on SSL cert verification, see https://github.com/ScienceCommons/api/issues/240

After some research I believe the root cause of the issue is that the SSL certificate for search.curatescience.org is [expired](https://www.sslshopper.com/ssl-checker.html#hostname=search.curatescience.org). It doesn't break in staging because I already updated the cert for elasticsearch-staging.curatescience.org

I do not have SSH access to the search.curatescience.org server, which is an AWS EC2 instance. @bcoe do you still have the .pem certificate and passphrase for this server? If so, can you provide them to me and/or @eplebel please?
